File: 02_processing_data/importing_data/csv_data.py
# - Import CSV Files ----
import pandas as pd # import the module 

# set working directory 
# os.getcwd() ~ get the current directory...
# os.chdir() ~ set the working directory...

df = pd.read_csv (r'./data/text.csv')
print(df.head) 
string_list = c(df[,'text'])
for i in range(len(string_list)):
  string_list[i] = string_list[i].lower()
print(string_list)

File: 02_processing_data/importing_data/csv_data.r
#install.packages('readr')
library(readr) # load the package into the r enviroment...
dir = getwd() # locate and store the set working directory of the current r session
setwd(dir) # confirm the directory is set to the working directory...

# Import the CSV Data ----
hiccups = read_csv("./data/Hiccups.csv")
str(hiccups) # Peak into the Structure of the Data,,,